pyxnat/core/interfaces.py

```diff
@@ -484,7 +484,11 @@ def load_config(self, location):
             raise Exception('Configuration file does not exist.')
 
     def version(self):
-        return self._exec('/data/version')
+        from pyxnat.core.errors import DatabaseError
+        try:
+            return self._exec('/data/version')
+        except DatabaseError:
+            return json.loads(self._exec('/xapi/siteConfig/buildInfo').decode())
```

Comment:
Bear in mind that this change modifies the type of object (string VERSUS dict) this function will return, however I am in favor of returning the whole `buildInfo` content.

Reply:
The type has changed because the data returned by the REST call has changed. What can we do? Casting the `dict` to `str` is not an option :wink:. I would leave it like this.